index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
- [ ] As stated by the previous reviewer, please add a class to the `h1` tag and reference the CSS style using the class name. This is done as shown below;

`<h1 class="heading">Hello Microverse</h1>`

- You can then reference it in CSS as shown below;

**.heading {
  color: blue;
}**